src/proto/Binance.proto

```diff
@@ -75,6 +75,40 @@ message TokenUnfreezeOrder {
     int64 amount = 3;  // amount of token to unfreeze
 }
 
+message HTLTOrder {
+    // 0xB33F9A24      // prefix
+    bytes from = 1;    // signer address
+    bytes to = 2;    // recipient address
+    string recipient_other_chain = 3;
+    string sender_other_chain = 4;
+    bytes random_number_hash = 5;
+    int64 timestamp = 6;
+    repeated SendOrder.Token amount = 7;
+    string expected_income = 8;
```

Comment:
`expected_income` what does this property mean?

Reply:
Please refer to the newest document: https://github.com/binance-chain/BEPs/blob/ea176af810196c21d6b5567ff7090c6247946ff0/BEP3.md

Reply:
Can you add this as a comment? 

> expected gained token on the other chain